src/datumaro/components/importer.py

```diff
@@ -197,7 +196,7 @@ def _change_context_root_path(context: FormatDetectionContext, path: str):
                 )
 
             for sub_dir in os.listdir(path):
-                if sub_dir.lower() in SUBSET_NAME_BLACKLIST:
+                if sub_dir.lower() not in ("train", "test", "val"):
```

Comment:
I meant what is the reason that we should remove `SUBSET_NAME_BLACKLIST`. Is it impossible such as

```suggestion
from datumaro.util.definitions import SUBSET_NAME_BLACKLIST, SUBSET_NAME_WHITELIST
# SUBSET_NAME_WHITELIST = ("train", "test", "val")

                if sub_dir.lower() in SUBSET_NAME_BLACKLIST or sub_dir.lower() not in SUBSET_NAME_WHITELIST:
```

p.s. I saw `("train", "test", "val")` is used other place so it seems possible to share it by declaring it at `datumaro.util.definitions`